### DISCUSSION TITLE
Skip data module same as CoreData?

### DISCUSSION THREAD

**Discussion by sal562** [skiptools/skip#59] [General]
Hi all is coredata supported at this time ? I saw https://github.com/skiptools/skip-data
But not sure if it’s the same thing.
Maybe transpiled to room or something?

**Comment by marcprux** [comment]
The closest thing we have right now it the SkipSQL module (https://github.com/skiptools/skip-sql). It is lower-level than Core Data/SwiftData/Room, but it can be the used as a foundation for local persistence. We have a sample app for it at https://github.com/skiptools/skipapp-databake

We hope to build on top of this in the future for a higher-level ORM solution.

**Reply by aehlke** [reply]
Any updates on plans for higher-level ORM?

**Reply by aabewhite** [reply]
It is still something we'd like to have, but we have not begun work on anything
